app/jobs/publisher_removal_job.rb

```diff
@@ -0,0 +1,26 @@
+class PublisherRemovalJob < ApplicationJob
+  queue_as :default
+
+  def perform(publisher_id:)
+    publisher = Publisher.find_by(id: publisher_id)
+    return if publisher.suspended?
+    publisher.status_updates.create(status: PublisherStatusUpdate::DELETED)
+    ActiveRecord::Base.transaction do
+      publisher.update(email: nil)
```

Comment:
Are there any view problems will having these be nil?

Reply:
There shouldn't be any problems given this will only render on admin.